playground/src/markdown/Explainer.md

```diff
@@ -102,11 +104,21 @@ Want to know more? [Here](https://wiki.polygon.technology/docs/miden/user_docs/a
 
 ## OK, but what can I do now?
 
-![](https://i.imgur.com/Y6pkjzt.png)
+![image](https://github.com/gubloon/examples/assets/7201911/1ef88c9a-1dab-48b5-9cd6-bf58c4b7aea4)
```

Comment:
I think we need a different picture here now. Those buttons don't exist anymore. It looks like

![Screenshot 2024-02-23 at 16 38 08](https://github.com/0xPolygonMiden/examples/assets/35031754/0acf3354-5d92-4579-a686-c5315566e03e)

Reply:
And then 
- debug is only there on Desktop (not mobile)
- verify is in the proof info box (on mobile it is there instead of debug)

Reply:
1. I replaced this image:
https://github.com/gubloon/examples/assets/7201911/1ef88c9a-1dab-48b5-9cd6-bf58c4b7aea4
with images for each section highlighting the relevant buttons according to the section.
2. Added descriptions in italic for both debug button and verify button as how they will appear in desktop and mobile.

Reply:
nice thanks, then let's merge